environment-dev.yml

```
@@ -21,3 +21,5 @@ dependencies:
       - mkdocs-gen-files
       - material-plausible-plugin
       - mktestdocs
+      # FIXME: replace this with a stable release when available
+      - mkdocs-gallery @ git+https://github.com/pmeier/mkdocs-gallery@async-gallery
```

My PR was merged and is available starting with

```suggestion
      - mkdocs-gallery >=0.10
```